### DISCUSSION TITLE
ZBBridge support for Honeywell Smoke Detector (JTYJ-GD-01LM/BW)

### DISCUSSION THREAD

**Discussion by Riza-Aslan** [arendst/Tasmota#11037] [Ideas]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
I've added some Xiaomi Honeywell Smoke Detectors to my ZBBridge and it receives the smokedensity, batterypercentage and linkquality, but it should also have a channel for "smoke" with a value of true or false. This doesn't show up while pairing the device.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [X] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [X] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [X] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [X] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [X] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [X] Device used (e.g., Sonoff Basic): Snoff ZBBridge
- [X] Tasmota binary firmware version number used: ZBBridge 9.2
  - [X] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Add a JTYJ-GD-01LM/BW smoke detector. The smoke channel doesn't show up in the initialization process

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by ascillato** [comment]
Please, could you be so kind on completing the [troubleshooting template](https://github.com/arendst/Tasmota/issues/new/choose) in order to have more information so as to properly help you? And also please provide the logs requested in the template. 

Remember to read the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md). Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by s-hadinger** [comment]
Especially, please provide `weblog 3` level logs when the device reports values.

**Comment by Riza-Aslan** [comment]
I hope these are the right lines from the log, because my ZbBridge is communication pretty much:

```
14:34:55 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":1280,"srcaddr":"0xA200","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":84,"securityuse":0,"seqnumber":98,"fc":"0x19","manuf":"0x0000","transact":2,"cmdid":"0x00","payload":"000000FF0000"}}
14:34:55 ZIG: ZbZCLRawReceived: {"0xA200":{"0500<00":"000000FF0000","ZoneStatusChange":0,"Endpoint":1,"LinkQuality":84}}
14:34:55 MQT: tele/Buero_Rauchmelder/SENSOR = {"ZbReceived":{"Buero_Rauchmelder":{"Device":"0xA200","Name":"Buero_Rauchmelder","0500<00":"000000FF0000","ZoneStatusChange":0,"Endpoint":1,"LinkQuality":84}}}
14:34:55 ZIG: {"ZbEZSPReceived":"4500000401000001010001000063B0C800A2FFFF2E18030A01FF422801210D0C03281C0421A80105215C00062402000000000A2100000821041064200196230000000002"}
14:34:55 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":0,"srcaddr":"0xA200","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":81,"securityuse":0,"seqnumber":99,"fc":"0x18","manuf":"0x0000","transact":3,"cmdid":"0x0A","payload":"01FF422801210D0C03281C0421A80105215C00062402000000000A21000008210410642001962300000000"}}
14:34:55 ZIG: ZbZCLRawReceived: {"0xA200":{"0000/FF01":"01210D0C03281C0421A80105215C00062402000000000A21000008210410642001962300000000","Endpoint":1,"LinkQuality":81}}
14:34:55 MQT: tele/Buero_Rauchmelder/SENSOR = {"ZbReceived":{"Buero_Rauchmelder":{"Device":"0xA200","Name":"Buero_Rauchmelder","BatteryVoltage":3.09,"BatteryPercentage":100,"SmokeDensity":1,"Endpoint":1,"LinkQuality":81}}}
```

**Reply by s-hadinger** [reply]
Thanks. This is useful. Can you also provide the result of `ZbInfo 0xA200`

**Reply by Riza-Aslan** [reply]
Sure:

```
14:46:52 CMD: ZbInfo 0xA200
14:46:52 SRC: WebConsole from 10.14.8.151
14:46:52 CMD: Group 0, Index 1, Command "ZBINFO", Data "0xA200"
14:46:52 MQT: tele/Buero_Rauchmelder/SENSOR = {"ZbInfo":{"Buero_Rauchmelder":{"Device":"0xA200","Name":"Buero_Rauchmelder","IEEEAddr":"0x00158D0002B2246C","ModelId":"lumi.sensor_smoke","Manufacturer":"LUMI","Endpoints":[1],"Config":["I01","A01.3"],"ZoneType":40,"ZoneStatus":0,"Reachable":true,"BatteryPercentage":100,"LastSeen":468,"LastSeenEpoch":1614343144,"LinkQuality":0}}}
14:46:52 MQT: stat/ZigBee2Tasmota/RESULT = {"ZbInfo":"Done"}
```

**Reply by s-hadinger** [reply]
Thanks. What version of Tasmota are you using?

**Reply by s-hadinger** [reply]
I see 9.2 above. Please try 9.3, it should be fixed in 9.3

**Reply by Riza-Aslan** [reply]
The pre compiled tasmota-zbbridge.bin release 9.2.0
I've just seen that there is a new release. Should I update to 9.3.1 first?

**Reply by Riza-Aslan** [reply]
Okay...great. But should I really take 9.3 or better 9.3.1?

**Comment by Riza-Aslan** [comment]
I've just updated to 9.31 now


`ZbInfo 0xA200` tells me now:
```
15:06:43.190 CMD: Group 0, Index 1, Command "ZBINFO", Data "0xA200"
15:06:43.199 MQT: tele/ZigBee2Tasmota/Buero_Rauchmelder/SENSOR = {"ZbInfo":{"Buero_Rauchmelder":{"Device":"0xA200","Name":"Buero_Rauchmelder","IEEEAddr":"0x00158D0002B2246C","ModelId":"lumi.sensor_smoke","Manufacturer":"LUMI","Endpoints":[1],"Config":["I01","A01.3"],"ZoneType":40,"ZoneStatus":0,"Reachable":true,"BatteryPercentage":100,"LastSeen":54,"LastSeenEpoch":1614344749,"LinkQuality":0}}}
15:06:43.204 MQT: stat/ZigBee2Tasmota/RESULT = {"ZbInfo":"Done"}
```

If I push the button, I'm getting the following log entry:
```
15:05:49.539 ZIG: ZbZCLRawReceived: {"0xA200":{"0500<00":"000000FF0000","ZoneStatusChange":0,"0500/FFF3":0,"Endpoint":1,"LinkQuality":0}}
15:05:49.545 MQT: tele/ZigBee2Tasmota/Buero_Rauchmelder/SENSOR = {"ZbReceived":{"Buero_Rauchmelder":{"Device":"0xA200","Name":"Buero_Rauchmelder","0500<00":"000000FF0000","ZoneStatusChange":0,"Fire":0,"Endpoint":1,"LinkQuality":0}}}
```

I think that "Fire" should change to 1 if there is smoke detected, right?

**Reply by s-hadinger** [reply]
Yes. It is registered as a fire detector.

**Reply by Riza-Aslan** [reply]
Awesome. Thanks for your help!
